src/main/java/at/hannibal2/skyhanni/utils/HypixelCommands.kt

```diff
@@ -150,6 +150,10 @@ object HypixelCommands {
         send("chatprompt $prompt")
     }
 
+    fun cb(uuid: String) {
```

Comment:
Please rename the function to callback. See the functions `bazaar()` or `auctionHouse()` in this file.